### PR TITLE
fix: ContextualMenuSplitButton always sets aria-checked (does not leave it undefined) when it is checkable

### LINKED ISSUE #27594: [Bug]: FluentUI ContextualMenu Accessibility Issue aria-requiered-attr
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Package version: @fluentui/react: 8.93.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu

## Repro Steps
- Navigate to the documentation for the ContextualMenu component: https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu
- Find the "ContextualMenu with checkable menu items and toggleable split button" example
- Open the contextual menu
- Run the Accessibility Insights for Web fast pass.

## Actual Behavior
Implementations of the FluentUI ContextualMenu are currently failing the Accessibility Insights for Web fast pass. See screenshots and attached logs.

image
image

## Expected Behavior
The ContextualMenu component should pass the Accessibility Insights for Web fast pass without any accessibility issues.

### Logs

```shell
Title: WCAG 4.1.2: Ensures elements with ARIA roles have all required ARIA attributes (.splitContainer-422[aria-posinset="6"][role="menuitemcheckbox"])
Tags: Accessibility, WCAG 4.1.2, aria-required-attr

Issue: Ensures elements with ARIA roles have all required ARIA attributes (aria-required-attr - https://accessibilityinsights.io/info-examples/web/aria-required-attr)

Target application: Fluent UI - Controls - React - ContextualMenu - https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu

Element path: .splitContainer-422[aria-posinset="6"][role="menuitemcheckbox"]

Snippet: <div role="menuitemcheckbox" class="splitContainer-422" aria-disabled="false" aria-expanded="false" aria-haspopup="true" aria-posinset="6" aria-setsize="10" tabindex="-1" data-is-focusable="true">

How to fix: 
Fix any of the following:
  Required ARIA attribute not present: aria-checked

Environment: Microsoft Edge version 112.0.1722.39

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
```

### Requested priority

Blocking

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by khmakoto**
## Previous Behavior

Checkable split buttons inside of contextual menus are not setting `aria-checked` when neither `isChecked` nor `checked` are passed in. `aria-checked` should be set to `false` if neither `isChecked` nor `checked` are passed in.

## New Behavior

Checkable split buttons inside of contextual menus now set `aria-checked` to `false` when neither `isChecked` nor `checked` are passed in. We're also now using the `getIsChecked` function to determine that since that is what is used at the end to determine if the split button is checkable or not.

## Related Issue(s)

- Fixes #27594

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/bb8dfe06f4f2910eef1561c76410bf91b009fa37'>bb8dfe06f4f2910eef1561c76410bf91b009fa37</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-8279ee8e-b6e5-4f7f-b9e7-18a13d31e6cb.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-8279ee8e-b6e5-4f7f-b9e7-18a13d31e6cb.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27650&commitId=533b2d339fb5d2b6224449a60ffa2eae65e9804a&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Review by smhigley — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/BaseButton-mount_base.html">792</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/BaseButton-mount.html">773</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Breadcrumb-mount_base.html">2028</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Breadcrumb-mount.html">2032</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Checkbox-mount_base.html">2083</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Checkbox-mount.html">2102</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/CheckboxBase-mount_base.html">1854</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/CheckboxBase-mount.html">1857</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ChoiceGroup-mount_base.html">3503</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ChoiceGroup-mount.html">3499</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ComboBox-mount_base.html">796</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ComboBox-mount.html">803</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/CommandBar-mount_base.html">7645</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/CommandBar-mount.html">7693</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ContextualMenu-mount_base.html">12852</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ContextualMenu-mount.html">13058</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/DefaultButton-mount_base.html">948</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/DefaultButton-mount.html">926</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/DetailsRow-mount_base.html">2620</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/DetailsRow-mount.html">2617</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/DetailsRowFast-mount_base.html">2642</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/DetailsRowFast-mount.html">2609</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/DetailsRowNoStyles-mount_base.html">2439</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/DetailsRowNoStyles-mount.html">2450</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Dialog-mount_base.html">3213</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Dialog-mount.html">3232</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/DocumentCardTitle-mount_base.html">298</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/DocumentCardTitle-mount.html">285</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Dropdown-mount_base.html">2451</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Dropdown-mount.html">2451</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/FocusTrapZone-mount_base.html">1431</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/FocusTrapZone-mount.html">1394</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/FocusZone-mount_base.html">1413</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/FocusZone-mount.html">1347</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedList-mount_base.html">41293</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedList-mount.html">45595</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedList-virtual-rerender_base.html">22016</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedList-virtual-rerender.html">22104</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">62606</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedList-virtual-rerender-with-unmount.html">62863</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedListV2-mount_base.html">284</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedListV2-mount.html">295</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedListV2-virtual-rerender_base.html">264</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedListV2-virtual-rerender.html">272</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">280</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">292</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/IconButton-mount_base.html">1298</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/IconButton-mount.html">1299</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Label-mount_base.html">446</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Label-mount.html">443</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Layer-mount_base.html">3359</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Layer-mount.html">3358</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Link-mount_base.html">520</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Link-mount.html">530</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/MenuButton-mount_base.html">1140</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/MenuButton-mount.html">1148</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/MessageBar-mount_base.html">25839</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/MessageBar-mount.html">25980</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Nav-mount_base.html">2334</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Nav-mount.html">2336</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/OverflowSet-mount_base.html">1022</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/OverflowSet-mount.html">997</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Panel-mount_base.html">2167</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Panel-mount.html">2129</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Persona-mount_base.html">921</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Persona-mount.html">892</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Pivot-mount_base.html">1067</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Pivot-mount.html">1095</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/PrimaryButton-mount_base.html">1044</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/PrimaryButton-mount.html">1057</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Rating-mount_base.html">5783</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Rating-mount.html">5797</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/SearchBox-mount_base.html">1115</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/SearchBox-mount.html">1110</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Shimmer-mount_base.html">2340</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Shimmer-mount.html">2306</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Slider-mount_base.html">1674</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Slider-mount.html">1678</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/SpinButton-mount_base.html">3421</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/SpinButton-mount.html">3418</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Spinner-mount_base.html">519</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Spinner-mount.html">520</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/SplitButton-mount_base.html">2142</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/SplitButton-mount.html">2157</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Stack-mount_base.html">541</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Stack-mount.html">540</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/StackWithIntrinsicChildren-mount_base.html">1084</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/StackWithIntrinsicChildren-mount.html">1079</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/StackWithTextChildren-mount_base.html">3371</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/StackWithTextChildren-mount.html">3391</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/SwatchColorPicker-mount_base.html">7447</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/SwatchColorPicker-mount.html">7417</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/TagPicker-mount_base.html">1681</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/TagPicker-mount.html">1695</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Text-mount_base.html">490</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Text-mount.html">490</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/TextField-mount_base.html">1150</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/TextField-mount.html">1142</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ThemeProvider-mount_base.html">1016</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ThemeProvider-mount.html">1019</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ThemeProvider-virtual-rerender_base.html">796</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ThemeProvider-virtual-rerender.html">783</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1557</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1563</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Toggle-mount_base.html">782</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/Toggle-mount.html">793</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/buttonNative-mount_base.html">254</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27650/perf-test/buttonNative-mount.html">240</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-8279ee8e-b6e5-4f7f-b9e7-18a13d31e6cb.json (hunk 1 above)
Comment (khmakoto) [reply change/@fluentui-react-8279ee8e-b6e5-4f7f-b9e7-18a13d31e6cb.json:1]: These two tests are flaky + not related to the changes in this PR.

**Pull request merged**